rs/bitcoin/ckbtc/minter/src/lifecycle/upgrade.rs

```diff
@@ -45,14 +46,14 @@ pub struct UpgradeArgs {
     pub kyt_principal: Option<CanisterId>,
 }
 
-pub fn post_upgrade(upgrade_args: Option<UpgradeArgs>) {
+pub fn post_upgrade<R: CanisterRuntime>(upgrade_args: Option<UpgradeArgs>, runtime: &R) {
     if let Some(upgrade_args) = upgrade_args {
         log!(
             P0,
             "[upgrade]: updating configuration with {:?}",
             upgrade_args
         );
-        record_event(&Event::Upgrade(upgrade_args));
+        record_event(EventType::Upgrade(upgrade_args), runtime);
```

Comment:
It looks like only main.rs calls `post_upgrade`, so why not just use `IC_CANISTER_RUNTIME` here instead of taking a runtime parameter?

Reply:
Good point. Done!